website/docs/api/Humanizer.IGrammaticalCaseTimeSpanFormatter.md

```
@@ -0,0 +1,54 @@
+### Methods
+
+<a name='Humanizer.IGrammaticalCaseTimeSpanFormatter.TimeSpanHumanize(Humanizer.TimeUnit,int,Humanizer.GrammaticalCase)'></a>
+
+## IGrammaticalCaseTimeSpanFormatter\.TimeSpanHumanize\(TimeUnit, int, GrammaticalCase\) Method
+
+Returns the locale\-authored unit\-case phrase for a duration count\.
+
+```csharp
+string TimeSpanHumanize(Humanizer.TimeUnit timeUnit, int unit, Humanizer.GrammaticalCase grammaticalCase);
+```
+#### Parameters
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)